ui/components/ui/update-nickname-popover/update-nickname-popover.stories.js

```diff
@@ -1,44 +1,70 @@
-import React, { useState } from 'react';
-import { action } from '@storybook/addon-actions';
-import { text } from '@storybook/addon-knobs';
+import { useArgs } from '@storybook/client-api';
+import React from 'react';
 import Button from '../button';
+import README from './README.mdx';
 import UpdateNicknamePopover from '.';
 
 export default {
   title: 'Components/UI/UpdateNickname',
   id: __filename,
+  parameters: {
+    docs: {
+      page: README,
+    },
+  },
+  argTypes: {
+    address: {
+      control: { type: 'text' },
+      defaultValue: '0xdeDbcA0156308960E3bBa2f5a273E72179940788',
+    },
+    showPopover: {
+      control: { type: 'boolean' },
+      defaultValue: false,
+    },
+    onAdd: { action: 'onAdd' },
```

Comment:
Even though we are using `useArgs` to handle `onClose` and the action won't show up in the actions panel(Which IMO it would be awesome if it did) I still think there is value in adding it here so it shows up in the controls panel as a function control.
```suggestion
    onAdd: { action: 'onAdd' },
    onClose: { action: 'onClose' },
```